### ISSUE TITLE
Question: Does this work with test.describe in playwright?

### ISSUE THREAD

**Issue by itz4blitz** [niche-tester/playwright-testrail-reporter#4]
Does mapping work with test.describe? We have several test files with test describes added so they can utilize different storage states without having to create a new browser context and storage state inside of the test itself. Would mapping still work in this example?

**Comment by denisha** [comment]
I have not looked into that, but will test it out when i have some time and get back to you

**Comment by denisha** [comment]
1. Can you provide a summary of your use case i.e. the intention behind using a case id in the test.describe instead of on the test level  
2.  What configuration are you using for your project on TestRail? :

<img width="813" alt="Screenshot 2023-05-05 at 15 37 03" src="https://user-images.githubusercontent.com/1721580/236580370-76f412c8-bb94-4de7-837f-7f230b0b8f2a.png">

**Comment by itz4blitz** [comment]
Hi @denisha

> Can you provide a summary of your use case i.e. the intention behind using a case id in the test.describe instead of on the test level
In some of our test files we need to use multiple storageState.json files in playwright. Rather than using a browser fixture and creating a new context and applying a storage state to the browser, and then creating a page, we find it is cleaner when we primarily need a single storage state to use `test.use({ storageState: './path/to/auth.json')` in our Playwright test. Without using test.describe's to surround our test(s) that need that 1 specific storage state, we are restricted to using either the browser approach I mentioned previously, or only using test.use() once for all tests in that test file. 

>What configuration are you using for your project on TestRail?
I'm not an admin in our Test Rail tenant, so I could be wrong, but I believe we use the single repository for all cases. 

I did want to mention, so far this npm package is working great for our team. We've been able to create runs and better track the status of our automated tests, so thank you for your work!
